app/services/stats/base_appeals_performance_report.rb

```diff
@@ -15,10 +15,10 @@ class BaseAppealsPerformanceReport < BaseReport
         responsible:                     ''
     }
 
-    def initialize(period_start= Time.now.beginning_of_year, period_end=Time.now)
+    def initialize(period_start = nil, period_end = nil)
```

Comment:
`period_start` and `period_end` are not used in this method. And is there a reason to keep the commented-out assignments to `@period_start` and `@period_end` below?